### DISCUSSION TITLE
Error: Invalid value for "path" argument /not able to view the editor

### DISCUSSION THREAD

**Discussion by aaditya-cpu** [Budibase/budibase#2602] [Help] [answered]
`Uncaught (in promise) Error: Invalid value for "path" argument
    o http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:10
    Ws http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:10
    Ws http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:10
    Fe http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:1
    Xs http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:10
    bd http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:23
    fd http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:23
    Fe http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:1
    yd http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:23
    JT http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:294
    nk http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:294
    ik http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:294
    Fe http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:1
    sk http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:294
    rk http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:294
    tk http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:294
    ak http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:294
    p http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:1
    zl http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:10
    Fe http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:1
    layout http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:23
    JT http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:294
    nk http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:294
    ik http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:294
    Fe http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:1
    sk http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:294
    CM http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:294
    OM http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:294
    p http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:1
    xM http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:294
    p http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:1
    ld http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:23
    Fe http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:1
    ud http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:23
    _M http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:294
    Fe http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:1
    SM http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:294
    RM http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:294
    p http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:1
    bM http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:294
    p http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:1
    ld http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:23
    Fe http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:1
    ud http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:23
    fM http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:294
    Fe http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:1
    yM http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:294
    zM http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:294
    EM http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:294
    p http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:294
    xe http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:1
    ye http://localhost:10000/prod-budi-app-assets/app_c83b11f500094b1baf6115592e66b68c/budibase-client.js:1
budibase-client.js:10:87103
`
![image](https://user-images.githubusercontent.com/72136610/133209570-6f47ecfc-5dd8-485a-a2e2-421eafda8fa4.png)

**Comment by mjashanks** [comment, marked as answer]
Hey @aaditya-cpu 

Does this happen with all your screens, or just `/ci_bank`? 

If you are able to export your app and email me the file (address in my profile), I could debug this. Note that the export will contain any data.

It could also be worth trying exporting and re-importing your app.

**Reply by aaditya-cpu** [reply]
All the screens dear one, 
Please check the mail.

**Reply by mjashanks** [reply]
Hey @aaditya-cpu - the issue is because of you `Main` screen. The route is set to `Main` - it should be `/Main`. Routes must begin with a `/`.

Changing the route to `/Main`, fixes your app.

We should handle this better - logged an issue: https://github.com/Budibase/budibase/issues/2611. Thanks for reporting!

**Reply by aaditya-cpu** [reply]
Is it possible to fix it for the  time being form my end?
how do i change the route?

**Reply by mjashanks** [reply]
Click on your screen and type into the `Route` input, on the right-hand settings panel:
![image](https://user-images.githubusercontent.com/3524181/133429662-cd801e40-6195-419f-b625-f127c6472b68.png)

**Reply by aaditya-cpu** [reply]
That did the trick Thanks

**Comment by aaditya-cpu** [comment]
How do I fix it mate? If I am to fix it locally  Sent from Mail for Windows From: Michael ShanksSent: 15 September 2021 02:37 PMTo: Budibase/budibaseCc: aaditya-cpu; MentionSubject: Re: [Budibase/budibase] Error: Invalid value for "path" argument /not able to view the editor (Discussion #2602) Hey @aaditya-cpu - the issue is because of you Main screen. The route is set to Main - it should be /Main. Routes must begin with a /.Changing the route to /Main, fixes your app.We should handle this better - I will log an issue, thanks for reporting!—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.
